scripts/deploy/v2Phase1-part1.ts

```diff
@@ -0,0 +1,162 @@
+const MIN_LBP_SIZE: string = toBN(100000).mul(ethers.constants.WeiPerEther).toString(); // 100k FEI
+
+// PCV Equity Minter
+const PCV_EQUITY_MINTER_INCENTIVE: string = toBN(1000).mul(ethers.constants.WeiPerEther).toString(); // 1000 FEI
+const PCV_EQUITY_MINTER_FREQUENCY = '604800'; // weekly
+const PCV_EQUITY_MINTER_APR_BPS = '1000'; // 10%
+
+// ERC20Splitter
+const SPLIT_DAO_BPS = '2000'; // 20%
+const SPLIT_LM_BPS = '2000'; // 20%
+const SPLIT_BURN_BPS = '6000'; // 60%
+
+// ETH Bonding Curve
+const SCALE: string = toBN(1).mul(ethers.constants.WeiPerEther).toString(); // 1 FEI
+const BUFFER = '50'; // 0.5%
+const DISCOUNT = '0'; // 0%
+const BC_DURATION = '86400'; // 1w
+const BC_INCENTIVE: string = toBN(500).mul(ethers.constants.WeiPerEther).toString();
+
+const USD_ADDRESS = '0x1111111111111111111111111111111111111111';
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  const {
+    core,
+    fei,
+    tribe,
+    feiEthPair,
+    sushiswapDpiFei,
+    weth,
+    chainlinkEthUsdOracleWrapper,
+    chainlinkRaiUsdCompositOracle,
+    chainlinkDaiUsdOracleWrapper,
+    compositeOracle,
+    aaveEthPCVDeposit,
+    compoundEthPCVDeposit,
+    chainlinkDpiUsdOracleWrapper,
+    daiBondingCurve,
+    dpiBondingCurve,
+    raiBondingCurve,
+    ethReserveStabilizer,
+    rariPool8FeiPCVDeposit,
+    rariPool7FeiPCVDeposit,
+    rariPool6FeiPCVDeposit,
+    rariPool9FeiPCVDeposit,
+    rariPool19DpiPCVDeposit,
+    rariPool19FeiPCVDeposit,
+    rariPool18FeiPCVDeposit,
+    rariPool24FeiPCVDeposit,
+    rariPool25FeiPCVDeposit,
+    rariPool26FeiPCVDeposit,
+    rariPool27FeiPCVDeposit,
+    rariPool9RaiPCVDeposit,
+    creamFeiPCVDeposit,
+    compoundDaiPCVDeposit,
+    aaveRaiPCVDeposit,
+    ethLidoPCVDeposit,
+    dai,
+    dpi,
+    rai,
+    proxyAdmin,
+    erc20Dripper,
+    balancerLBPoolFactory
+  } = addresses;
+
+  const {
+    uniswapRouter: uniswapRouterAddress,
+    sushiswapRouter: sushiswapRouterAddress,
+    chainlinkTribeEthOracle: chainlinkTribeEthOracleAddress
+  } = getAllContractAddresses();
+
+  if (!core || !feiEthPair || !weth || !uniswapRouterAddress || !chainlinkEthUsdOracleWrapper || !compositeOracle) {
+    console.log(`core: ${core}`);
+    console.log(`feiEtiPair: ${feiEthPair}`);
+    console.log(`weth: ${weth}`);
+    console.log(`uniswapRouter: ${uniswapRouterAddress}`);
+    console.log(`chainlinkEthUsdOracleWrapper: ${chainlinkEthUsdOracleWrapper}`);
+    console.log(`compositeOracle: ${compositeOracle}`);
+
+    throw new Error('An environment variable contract address is not set');
+  }
+
+  // ----------- Replacement Contracts ---------------
+  const uniswapPCVDepositFactory = await ethers.getContractFactory('UniswapPCVDeposit');
+  const uniswapPCVDeposit = await uniswapPCVDepositFactory.deploy(
+    core,
+    feiEthPair,
+    uniswapRouterAddress,
+    chainlinkEthUsdOracleWrapper,
+    ethers.constants.AddressZero,
+    '100'
```

Comment:
What is this 100? Should be a constant

Reply:
You wrote this ;)

But yes I will change to a constant.